### ISSUE TITLE
bumping svg version in dependencies

### ISSUE THREAD

**Issue by WeepingClown13** [Canop/csv2svg#5]
Hi, first of all, thanks for making csv2svg.

I have packaged csv2svg for Debian, but since the version of the svg package in Debian currently is 0.16.0, I had to bump that dependency version. However, since svg version 0.16.0 takes a content parameter for the `element::Text::new()` method, the build has been failing. The content param is of a  generic type that have an `Into<String>` implementation, and thus using a simple `""` could solve the issue without affecting anything at all. The following patch can fix the issue after the `svg = 0.16` bump;

```bash
--- a/src/graph.rs
+++ b/src/graph.rs
@@ -57,7 +57,7 @@
                 .set("height", 8)
                 .set("fill", COLORS[seq_idx]);
             group.append(square);
-            let label = element::Text::new()
+            let label = element::Text::new("")
                 .set("x", x + 14)
                 .set("y", y + 7)
                 .set("fill", LEGEND_COLOR)
@@ -83,7 +83,7 @@
                 .set("stroke-dasharray", "10 7")
                 .set("d", data);
             group.append(path);
-            let tick_label = element::Text::new()
+            let tick_label = element::Text::new("")
                 .set("x", self.gr.left + self.gr.width + 2)
                 .set("y", self.projector.project_y(*tick) + 2)
                 .set("fill", TICK_LABEL_COLOR)
@@ -165,7 +165,7 @@
                 .set("opacity", 0.5)
                 .set("d", data);
             tick_opt_group.append(path);
-            let tick_label = element::Text::new()
+            let tick_label = element::Text::new("")
                 .set("x", tick.tx + 1)
                 .set("y", y + 9)
                 .set("fill", TICK_LABEL_COLOR)
@@ -215,7 +215,7 @@
                             .set("r", 8);
                         point_group.append(circle);
                         let mut point_opt_group = node::element::Group::new().set("class", "opt");
-                        let point_label_shadow = element::Text::new()
+                        let point_label_shadow = element::Text::new("")
                             .set("x", x - 5)
                             .set("y", y - 10)
                             .set("stroke", "#222")
@@ -230,7 +230,7 @@
                             .set("cy", y)
                             .set("r", 4);
                         point_opt_group.append(circle);
-                        let point_label = element::Text::new()
+                        let point_label = element::Text::new("")
                             .set("x", x - 5)
                             .set("y", y - 10)
                             .set("fill", TICK_LABEL_COLOR)
```

If you find this acceptable, I can open a pr with the same and bumping the svg version to 0.16. Having this change upstream would help us from maintaining two patches downstream. So, can you consider this dependency bump?

**Comment by Canop** [comment]
> If you find this acceptable, I can open a pr with the same and bumping the svg version to 0.16.

Sorry, I wanted to have a look at the consequences, so I did the PR myself.
I'll wait for you to confirm it looks OK before merging it.

**Comment by WeepingClown13** [comment]
I am fine with the changes. My primary need is the svg version bump anyway. You can go ahead with your preferred changes and I'll be satisfied as long as the svg version is bumped. Thanks for working on this :)

**Comment by Canop** [comment]
Released 0.2.2
